docs/release-notes.md

```diff
@@ -2,51 +2,34 @@
-- **API refactoring is complete:** In 1.2 we refactored the RPS REST APIs and in 1.3 we performed a similar refactoring the MPS REST APIs.  With the release of 1.3, these API breaking changes have been completed.  These changes align our APIs to industry standards for RESTful interfaces, normalizes responses, and simplifies the layout into logical groupings.  Special note: with the addition of Stateless Authentication, the API key in the header has been removed.  Please take a look at our updated Swagger documentation: 
-    * [RPS on SwaggerHub](https://app.swaggerhub.com/apis-docs/rbheopenamt/rps/1.3.0#/)
-    * [MPS on SwaggerHub](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.3.0#/)
-#### RPC
-- **Added CA-Certs to dockerfile:** In order to support certificate checking when running in a docker container, we added the CA-Certs from the base Ubuntu image to the RPC container.  This small change bumps RPC to version 1.2.1.  RPC 1.2.1 is compatible with all other 1.2 and 1.3 Open AMT Cloud Toolkit components.
 
+#### MPS Router
+- **Reverse Proxy:** In this release we are adding a new required component to the toolkit deployment, the MPS Router.  This small containerized GO serivce acts as a reverse proxy between the API Gateway and MPS.  When Intel&reg; AMT devices make a CIRA connection with MPS, the MPS registers the device GUID and its MPS instance ID with the MPS Router.  All MPS API calls coming from the API Gateway are first sent to the MPS Router before being forwarded to the correct MPS instance.  As with all components of the Open AMT Cloud Toolkit, this service is open source and is located in the [MPS Router](https://github.com/open-amt-cloud-toolkit/mps-router) repository.
 
+### Modifications and Removals
+#### MPS
+- **AllowList:** As noted in the 1.3 release notes, the AllowList was not working as designed.  What it was suppose to do is to give the ability to only allow specified devices (GUIDs) to connect to MPS.  When we removed "developer mode" this broke the AllowList functionality.  In this release we have updated MPS to only allow devices to connect if they are listed in the MPS database and present the proper credentials.  If a device either is not listed in the MPS database or has the wrong credentials, MPS will refuse the connection.  Along with this change, we have improved the MPS Devices API to make it very easy to add, edit, and remove devices from the MPS database.  MPS API documentation can be found out on [Swagger](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.4.0#/)
```

Comment:
- **AllowList:** Allowlist enables administrators to specify which devices (GUIDs) can connect to MPS. Removing "developer mode" in a previous version broke Allowlist functionality. In this release, MPS AllowList functionality allows devices to connect if they are listed in the MPS database and present the proper credentials.  If a device either is not listed in the MPS database or has the wrong credentials, MPS will refuse the connection.  Along with this change, we have improved the MPS Devices API to make it very easy to add, edit, and remove devices from the MPS database.  MPS API documentation can be found out on [Swagger](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.4.0#/)

Reply:
Updated.